qiling/extensions/coverage/formats/ezcov.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+# 
+# Cross Platform and Multi Architecture Advanced Binary Emulation Framework
+#
+
+from os.path import basename
+
+from .base import QlBaseCoverage
+
+
+# Adapted from https://github.com/nccgroup/Cartographer/blob/main/EZCOV.md#coverage-data
+class bb_entry(dict):
+    def __init__(self, offset, size, mod_id):
```

Comment:
This class is being created in runtime. Probably it would be better to make it as fast as possible (to minimize performance penalty) by moving the calculations / string manipulations to the `csvline` method, which is called once after the execution has ended.